### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by hollymariegibbs**
No existing issue.

Changes in this pull request:
- Updated some spelling and grammar
- Added link to TypeScript documentation
- Clarified some language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 #Business
 C4GNV is a member brigade of Code for America, the goal of this project is to make it easier
 to start a business in Gainesville by providing people easy access to permits and zoning information.
-The project is a light weight website written in Typescript and Angular 2, using open data sources
-as a persistent backend.
+The project is a lightweight website written in [Typescript](https://www.typescriptlang.org/) and Angular 2, using open data sources as a persistent backend.
 
-The inspiration for this service is to eventually be able to expand to be like the
```
Comment (Bjorn248) [inline README.md:7]: Why remove "the" here? Comparing "to be like the Business Portal for San Francisco" and "to be like Business Portal for San Franscisco"...the second one just sounds wrong.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+The inspiration for this service is to eventually be able to expand to be like 
 [Business Portal for San Francisco](https://businessportal.sfgov.org/), however we're
-starting with just the [Permit Locator](https://businessportal.sfgov.org/permits-licenses) functionality.
+starting with the [Permit Locator](https://businessportal.sfgov.org/permits-licenses) functionality only.
```
Comment (Bjorn248) [inline README.md:8]: I'm not sure I understand this change, what's wrong with the original?

**Review by Bjorn248 — COMMENTED** [review]
Let's have a discussion about these changes.

File: README.md (hunk 1 above)
Comment (hollymariegibbs) [inline README.md:7]: the 'the' is fine, just unnecessary

File: README.md (hunk 2 above)
Comment (hollymariegibbs) [reply README.md:8]: 'just' is passive, moved placement for clarification

File: README.md (hunk 1 above)
Comment (Bjorn248) [reply README.md:7]: I think it reads strangely without "the" so if it's fine with it I'd rather keep it. The other changes seem fine though, so if you revert this change but keep the others we can merge them.

**Pull request merged**